backend/src/packages/projects/project.controller.ts

```diff
@@ -46,6 +48,15 @@ class ProjectController extends Controller {
       handler: () => this.findAll(),
     });
 
+    this.addRoute({
+      path: ProjectsApiPath.USER_ID,
+      method: 'GET',
+      handler: (options) =>
+        this.findByUserId(
+          options as ApiHandlerOptions<{ params: FindProjectByUserIdRequest }>,
```

Comment:
```suggestion
          options as ApiHandlerOptions<{ user: UserAuthResponse }>,
```
And route probably should be projects/my-projects or something like that